OSDOCS-11439: update microshift config defaults (#80598)
openshift/openshift-docs: modules/microshift-default-settings.adoc

  routeAdmissionPolicy:
    namespaceOwnership: InterNamespaceAllowed # <12>
  status: Managed # <13>
kubelet: # <14>
manifests: # <15>
  kustomizePaths:
    - /usr/lib/microshift/manifests
    - /usr/lib/microshift/manifests.d/*
    - /etc/microshift/manifests
    - /etc/microshift/manifests.d/*
network:
  clusterNetwork:
    - 10.42.0.0/16 # <16>
  serviceNetwork:
    - 10.43.0.0/16 # <17>
  serviceNodePortRange: 30000-32767 # <18>
node:
  hostnameOverride: "" # <19>
  nodeIP: "" # <20>
  nodeIPv6: "" # <21>

> storage:
>     driver: "" # <22>
>     optionalCsiComponents:
>         - "" # <23>

Reviewer: ```suggestion
storage:
  driver: "" # <22>
  optionalCsiComponents:  # <23>
  - ""
```

Reviewer: Moved the number since it's the `optionalCsiComponents` field that's actually being described below.

<11> Default ports shown. Configurable. Valid values for both port entries are a single, unique port in the 1-65535 range. The values of the `ports.http` and `ports.https` fields cannot be the same.
<12> Describes how hostname claims across namespaces are handled. By default, allows routes to claim different paths of the same hostname across namespaces. Valid values are `Strict` and `InterNamespaceAllowed`. Specifying `Strict` prevents routes in different namespaces from claiming the same hostname. If the value is deleted in a customized {microshift-short} `config.yaml`, the `InterNamespaceAllowed` value is automatically set.
<13> Default router status, can be `Managed` or `Removed`.

> <14> Optional parameter for configuring low latency. Default value is null.

Reviewer 2: ```suggestion
<14> Optional parameter for configuring kubelet. Default value is null.
```
Technically someone could configure anything related to kubelet using that field. We don't really want that, but saying it's for "configuring low latency" feels off.

> <14> Parameter for configuring kubelet. Used for low latency configuration. Default value is null.

Reviewer 3: ```suggestion
<14> Parameter for configuring kubelet. Used for low-latency configuration. Default value is null.
```

<15> The locations on the file system to scan for `kustomization` files to use to load manifests. Set to a list of paths to scan only those paths. Set to an empty list to disable loading manifests. The entries in the list can be glob patterns to match multiple subdirectories.
<16> A block of IP addresses from which pod IP addresses are allocated. This field is immutable after installation.

> <17> A block of virtual IP addresses for Kubernetes services. IP address pool for services. A single entry is supported. This field is immutable after installation.

Reviewer 4: These two now allow adding one more entry for the other IP family. It is true though that the first entry of the list must remain unchanged.

<18> The port range allowed for Kubernetes services of type `NodePort`. If not specified, the default range of 30000-32767 is used. Services without a `NodePort` specified are automatically allocated one from this range. This parameter can be updated after the cluster is installed.
<19> The name of the node. The default value is the hostname. If non-empty, this string is used to identify the node instead of the hostname.

> <20> The IP address of the node. The default value is the IP address of the default route.

Reviewer 5: Small nit. Maybe use IPv4 in place of IP here

<21> Configurable IPv6 network.
<22> Default value is empty. Valid values are "none" or "lvms". An empty value or null field defaults to LVMS deployment.

> <23> Default value is empty. An empty array defaults to deploying `snapshot-controller` and `snapshot-webhook`.Expected values are `csi-snapshot-controller`, `csi-snapshot-webhook`, or `none`. An entry of `none` is mutually exclusive with all other values.

Reviewer: ```suggestion
<23> Array. Default value is null or an empty array. A null or empty array defaults to deploying `snapshot-controller` and `snapshot-webhook`. Expected values are `csi-snapshot-controller`, `csi-snapshot-webhook`, or `none`. An entry of `none` is mutually exclusive with all other values.
```
